metaworld/envs/mujoco/sawyer_xyz/v2/sawyer_stick_pull_v2.py

```diff
@@ -49,26 +51,45 @@ def model_name(self):
 
     @_assert_task_is_set
     def step(self, action):
```

Comment:
You'll need to update this to the new `evaluate_state` method rather than `step` in order to be compatible with my mujoco exception-catching stuff.